numba/unicode.py

```diff
@@ -540,6 +540,93 @@ def find_impl(a, b):
         return find_impl
 
 
+@register_jitable
+def _check_type(ty, name, accepted=(types.NoneType,)):
+    """
+    Check object belongs to one of specified types
+    :param ty: object type
+    :param name: name of the argument
+    :param accepted: accepted types
+    """
+    thety = ty
+    # if the type is omitted, the concrete type is the value
+    if isinstance(ty, types.Omitted):
+        thety = ty.value
+    # if the type is optional, the concrete type is the captured type
+    elif isinstance(ty, types.Optional):
+        thety = ty.type
+
+    if thety is not None and not isinstance(thety, accepted):
+        raise TypingError('"{}" must be {}, not {}'.format(name, accepted, ty))
+
+
+@register_jitable
+def _adjust_start(length, start=0):
+    """
+    https://github.com/python/cpython/blob/201c8f79450628241574fba940e08107178dc3a5/Objects/unicodeobject.c#L9350-L9354
```

Comment:
These should probably be comments not docstrings.

Reply:
Ok, I will make the change.